core/src/epicli/data/common/ansible/playbooks/roles/repository/tasks/create-helm-repo.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Get list of system Helm charts directories
+  find:
+    paths: "{{ specification.apache_epirepo_path }}/helm-charts/system"
+    file_type: directory
+  register: system_charts_dir_list
+
+- name: Prepare repository server Helm charts
+  block:
+    - name: Generate index.yaml file
+      become: yes
+      shell: |- 
```

Comment:
Since `&&` is being used in the script, please use the `>-` here, this will remove extra new line characters.